databases/week3/databases_week3.sql

```diff
@@ -0,0 +1,209 @@
+CREATE database meal_sharing_db;
+USE meal_sharing_db;
+CREATE TABLE meal (
+id INT UNSIGNED NOT NULL AUTO_INCREMENT,
+title VARCHAR(255) NOT NULL,
+description TEXT NOT NULL,
+location VARCHAR(50) NOT NULL,
+`when` DATETIME NOT NULL,
+max_reservations INT NOT NULL,
+price DECIMAL(10,2) NOT NULL,
+created_date DATETIME NOT NULL DEFAULT NOW(),
+PRIMARY KEY (id)
+)ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE reservation (
+id INT UNSIGNED NOT NULL AUTO_INCREMENT,
+number_of_guests INT UNSIGNED NOT NULL,
+meal_id INT UNSIGNED NOT NULL,
+created_date DATETIME NOT NULL DEFAULT NOW(),
+PRIMARY KEY (id),
+FOREIGN KEY (meal_id) REFERENCES meal (id) ON DELETE CASCADE
+)ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE review (
+id INT UNSIGNED NOT NULL AUTO_INCREMENT,
+title VARCHAR(255) NOT NULL,
+description TEXT NOT NULL,
+meal_id INT UNSIGNED NOT NULL,
+stars INT UNSIGNED NOT NULL,
+created_date DATETIME NOT NULL DEFAULT NOW(),
+PRIMARY KEY (id),
+FOREIGN KEY (meal_id) REFERENCES meal (id) ON DELETE CASCADE
+)ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+-- CREATING queries
+-- MEAL
+-- Get all meals
+SELECT *
+FROM meal;
+-- Add a new meal
+
+INSERT INTO meal (
+title,
+description,
+location,
+`when`, 
+max_reservations,
+price
+)
+VALUES
+(
+'Hamburger', 
+'sandwich consisting of one or more cooked patties of ground meat, usually beef, placed inside a sliced bread roll or bun', 
+'Rådhuspladsen',
+'2021-04-26',
+4,
+80
+);
+-- Get a meal with any id, fx 1
+SELECT * 
+FROM meal
+WHERE id = 1;
+-- Update a meal with any id, fx 1. Update any attribute fx the title or multiple attributes
+UPDATE meal SET title = 'Ham cheese burger'
+WHERE meal.id = 1;
+
+ -- Delete a meal with any id, fx 1
+ 
+DELETE FROM meal
+WHERE id = 1;
+
+-- RESERVATION:
+
+-- Get all reservations
+
+SELECT * 
+FROM reservation;
+
+-- Add a new reservation
```

Comment:
This one crashes for me, can you fix it please? :)

Reply:
this is working for me

Reply:
So, the reason it crashes for me is because you set the `meal_id` to "2" but there are no entries in the meal table anymore. 
You should get the same error if you run the queries in the file in order.